modules/seqexec/engine/src/main/scala/seqexec/engine/Step.scala

```diff
@@ -115,16 +117,16 @@ object Step {
       */
     val toStep: Step[F] =
       Step(
-        id,
-        breakpoint,
-        Skipped(false),
-        skipMark,
-        done ++ List(focus.execution) ++ pending
+        id = id,
+        breakpoint = breakpoint,
+        skipped = Skipped(false),
+        skipMark = skipMark,
+        executions = done ++ focus.toParallelActionsList ++ pending
```

Comment:
Much easier to read now. 👍

Reply:
Yeah, I spent some time adding the field names or I'd get quite lost